src/uct/ib/base/ib_md.h

```diff
@@ -496,6 +489,12 @@ uct_ib_md_mem_dereg_params_invalidate_check(
     return UCS_OK;
 }
 
+static UCS_F_ALWAYS_INLINE uint8_t
+uct_ib_md_get_atomic_mr_id(uct_ib_md_t *md)
+{
+    return (uint8_t)(md->flush_rkey >> 8);
```

Comment:
1. do we really need an explicit cast to uint8_t
2. pls check clang-format output
3. seems used only once -> remove?